packages/editor/src/editor.spec.ts

```diff
@@ -199,6 +222,54 @@ describe('Editor', () => {
     expect(content).to.be.equal('1 + 1');
   });
 
-  // TODO: e2e or mock tests for _getExtension
-  // TODO: Write a node script that acts as an external editor and write e2e tests
+  context('runEditCommand', () => {
+    let externalEditor: string;
+
+    beforeEach(async() => {
+      externalEditor = await setupExternalEditor(base);
+    });
+
+    afterEach(async() => {
+      await fs.unlink(externalEditor);
```

Comment:
You could also make the `fs.rmdir` call above remove `base` instead of `tmpDir`, then we don’t need to worry about that here :)